projects/demo-integrations/cypress/tests/kit/date/date-basic.cy.ts

```diff
@@ -0,0 +1,201 @@
+import {DemoPath} from 'projects/demo/src/app/app.routes';
+
+describe('Date', () => {
```

Comment:
add tests for these cases, please:

* Empty input => type `9` => `09|`
* `27|` => type `9` => `27.09|`
* `3|` => Type `7` => no value changes